src/app/safe-delete-dialog/safe-delete-dialog.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Pls don't leave autogenerated stuff that you don't use. Delete .spec files unless you're going to write tests. Same applies for the .scss files.